reactivesocket-core/src/main/java/io/reactivesocket/Plugins.java

```diff
@@ -24,4 +25,30 @@
 
     private Plugins() {}
 
+    static {
+        if (Boolean.getBoolean("io.reactivesocket.fragmentation.enable")) {
+            int mtu = Integer.getInteger("io.reactivesocket.fragmentation.mtu", 1024);
+
+            if (Plugins.DUPLEX_CONNECTION_INTERCEPTOR == null) {
```

Comment:
Its static because are going to refactor setup and this was more for testing at moment